<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDiagnosticsDialog</class>
 <widget class="QDialog" name="ConfigDiagnosticsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2001</width>
    <height>794</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>380</y>
     <width>1091</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok|QDialogButtonBox::StandardButton::RestoreDefaults</set>
   </property>
  </widget>
  <widget class="QListView" name="lstCategory">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>191</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="wdgMainState" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>5</y>
     <width>881</width>
     <height>361</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection state:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblConnState">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>711</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid grey;</string>
    </property>
    <property name="text">
     <string>unknown</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Autobuzzer state:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAutoBuzzerState">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid grey;</string>
    </property>
    <property name="text">
     <string>unknown</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdTestRing">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Test ring</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdTestBuzzer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Test buzzer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Device start time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDeviceStartTime">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid grey;</string>
    </property>
    <property name="text">
     <string>unknown</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdUpdateStartTime">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdChangeAutoBuzz">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>A test ring will not trigger the auto buzzer.</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="wdgRawData" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>871</width>
     <height>361</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Raw data:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdUpdateRawData">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtRawData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>851</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207); border: 1px solid black;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::LineWrapMode::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="wdgSettings" native="true">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>10</y>
     <width>881</width>
     <height>351</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>WLAN SSID pattern:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtWlanPattern">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MQTT Broker Address:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBrokerAddr">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="wdgActionLog" native="true">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>420</y>
     <width>881</width>
     <height>361</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ring/Buzz log:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdUpdateHistory">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtActionLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>861</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207); border: 1px solid black;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::LineWrapMode::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDiagnosticsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDiagnosticsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
